package.json

```diff
@@ -13,17 +13,16 @@
     "@viem/anvil": "^0.0.6",
     "@vitest/coverage-v8": "^0.34.2",
     "@wagmi/cli": "^1.3.0",
+    "dprint": "^0.40.2",
     "ethers": "^5.7.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "rome": "^12.1.2",
     "typescript": "^5.0.4",
     "vitest": "^0.34.1",
     "wagmi": "^1.3.11"
   },
   "scripts": {
-    "format": "rome format . --write",
-    "lint": "rome check --apply-unsafe .",
+    "format": "dprint fmt",
```

Comment:
dprint has linting?

Reply:
Maybe I am naive to what all we were getting here. This will e.g. sort imports for us. Doesn't flag unused variables though. Not sure what else we had. Worth leaving?

Reply:
I don't think dprint has linting after some brief research, and linting is pretty good at finding issues. 

Might be worth checking out [Biome](https://biomejs.dev/) (Rome successor).

Reply:
hmm idk I am looking through here and I don't see a lot I am really concerned about? 🤔

Reply:
Linters rarely catch anything that would make it past a code review but they still help provide feedback in your editor

Reply:
added biome linter, thanks!